doc/nrf/security/tfm/index.rst

```diff
@@ -0,0 +1,42 @@
+.. _ug_tfm_index:
+
+Trusted Firmware-M in the |NCS|
+###############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Nordic Semiconductor recommends following `Platform Security Architecture (PSA)`_ for product development to ensure appropriate security implementation in IoT devices.
+PSA offers the `PSA Certified IoT Security Framework`_ for securing connected devices, which consists of four steps:
```

Comment:
Are these called steps (or something else) in the CSA spec? Cannot verify as the site does not open.

Reply:
Yes, these are called steps on that website.